src/metabase/driver/generic_sql.clj

```diff
@@ -85,6 +86,9 @@
     "Return a korma form appropriate for converting a Unix timestamp integer field or value to an proper SQL `Timestamp`.
      SECONDS-OR-MILLISECONDS refers to the resolution of the int in question and with be either `:seconds` or `:milliseconds`."))
 
+(extend-protocol jdbc/IResultSetReadColumn
+  (class (object-array []))
+  (result-set-read-column [x _2 _3] (PersistentVector/adopt x)))
```

Comment:
In Clojure arguments that are ignored are just called `_`. So this should be something like

```clojure
(result-set-read-column [x _ _] (PersistentVector/adopt x))
```